From 888e3d9c3bed2bd3e50d9350af8aa18c022374c5 Mon Sep 17 00:00:00 2001
From: Michel Thebeau <michel.thebeau@windriver.com>
Date: Wed, 20 Jun 2012 17:41:44 -0400
Subject: [PATCH] ocf: config OCF_TALITOS should depend on FSL_SOC &&
 !CRYPTO_DEV_TALITOS

Only boards with freescale Talitos hw should enable OCF_TALITOS.

OCF_TALITOS is expected to conflict with CRYPTO_DEV_TALITOS, so do not
allow it when CRYPTO_DEV_TALITOS is enabled.

Signed-off-by: Michel Thebeau <michel.thebeau@windriver.com>
---
 crypto/ocf/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/crypto/ocf/Kconfig b/crypto/ocf/Kconfig
index 65a4461..056fa1e 100644
--- a/crypto/ocf/Kconfig
+++ b/crypto/ocf/Kconfig
@@ -68,7 +68,7 @@ config OCF_HIFNHIPP
 
 config OCF_TALITOS
 	tristate "talitos (HW crypto engine)"
-	depends on OCF_OCF
+	depends on OCF_OCF && FSL_SOC && !CRYPTO_DEV_TALITOS
 	help
 	  OCF driver for Freescale's security engine (SEC/talitos).
 
-- 
1.7.9.7

